index.html

```diff
@@ -44,7 +44,7 @@
     <div class="shopping-cart">
       <p class="cart-title">Items in Cart</p>
       <div class="cart-list"></div>
-      <p>Total: $<span class="total">0</span></p>
+      <p style="font-weight: bold;">Total: $<span class="total">0</span></p>
```

Comment:
It's not terrible in this case, but in general inline styles are a bad idea because they're very hard to override.